cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,106 @@
+    <label class="label setting-label" for="list-license-types">
+        <%= gettext("License Type") %>
+    </label>
+    <ul class="license-types" id="list-license-types">
+        <% var link_start_tpl = '<a href="{url}" target="_blank">'; %>
+        <% _.each(licenseInfo, function(license, licenseType) { %>
+            <li data-license="<%- licenseType %>">
+                <button name="license-<%- licenseType %>"
+                        class="action license-button <% if(model.type === licenseType) { print("is-selected"); } %>"
+                        <% if (license.tooltip) { %>data-tooltip="<%- license.tooltip %>"<% } %>>
+                    <%- license.name %>
+                </button>
+                <p class="tip">
+                    <% if(license.url) { %>
+                        <a href="<%- license.url %>" target="_blank">
+                            <%= gettext("Learn more about {license_name}")
+                                        .replace("{license_name}", license.name)
+                            %>
+                        </a>
+                    <% } else { %>
+                        &nbsp;
+                    <% } %>
+                </p>
+            </li>
+        <% }) %>
+    </ul>
+
+<% var license = licenseInfo[model.type]; %>
+<% if(license && !_.isEmpty(license.options)) { %>
+    <div class="wrapper-license-options">
+        <label class="label setting-label" for="list-license-options">
+            <%- gettext("Options for {license_name}").replace("{license_name}", license.name) %>
+        </label>
+        <p class='tip tip-inline'>
+            <%- gettext("The following options are available for the {license_name} license.")
+                        .replace("{license_name}", license.name) %>
+        </p>
+        <ul class="license-options" id="list-license-options">
+            <% _.each(license.option_order, function(optionKey) { %>
+                <% var optionInfo = license.options[optionKey]; %>
+                <% if (optionInfo.type == "boolean") { %>
+                    <% var optionSelected = model.options[optionKey]; %>
+                    <% var optionDisabled = optionInfo.disabled ||
+                            (optionInfo.conflictsWith && _.any(optionInfo.conflictsWith, function(key) {return model.options[key];}));
+                    %>
+                    <li data-option="<%- optionKey %>"
+                            class="action-item license-button
+                                   <% if (optionSelected) { print("is-selected"); } %>
+                                   <% if (optionDisabled) { print("is-disabled"); } else { print("is-clickable"); } %>"
+                    >
+                        <i class="fa fa-fw
+                             <% if(optionSelected) { print("fa-check-square-o"); } else { print("fa-square-o"); } %>
+                             <% if(optionDisabled) { print("is-disabled"); } %>
+                             "></i>
```

Comment:
@singingwolfboy good question.  Since a screen reader user is likely to browse by Heading, out of context, it would be helpful to have the selected/disabled state available in the '''<h4>``` 
IMHO, all states can be in one selector.  A couple of things to consider: I try to avoid using unnecessary punctuation in SR text since it is read outloud i.e open-paren, close-paren.  You should also consider spacing.  There needs to be a trailing space ```&nbsp;``` for SR text that precedes other text and vice versa.  Screen readers will have a hard time reading "Attributionselecteddisabled".  Commas add a nice pause, consider 
```
<h4 class="option-name">Attribution<span class="sr">, selected, disabled</span></h4>
```